public/Algorithm/BinarySearch/leetcode/108. Convert Sorted Array to Binary Search Tree/claude sonnet 4.6 adaptive/README_react.html

```
@@ -0,0 +1,1680 @@
+
+                <pre class="line-numbers"><code class="language-python">from typing import Optional
+
+# LeetCode が提供する TreeNode クラス（定義済みとして扱う）
+# class TreeNode:
+#     def __init__(self, val=0, left=None, right=None):
+#         self.val = val
+#         self.left = left
+#         self.right = right
+
+class Solution:
+    def sortedArrayToBST(self, nums: list[int]) -> Optional["TreeNode"]:
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)